app/models/host/base.rb

```diff
@@ -94,7 +94,11 @@ def set_non_empty_values(parser, methods)
 
     def set_interfaces(parser)
       parser.interfaces.each do |name, attributes|
-        macaddress = Net::Validations.normalize_mac(attributes[:macaddress])
+        begin
+          macaddress = Net::Validations.normalize_mac(attributes[:macaddress])
+        rescue ArgumentError
+          logger.debug "invalid mac during parsing: #{attributes[:macaddress]}"
+        end
         base = self.interfaces.where(:mac => macaddress)
```

Comment:
this would match interfaces with nil mac which does not make sense, we should try to find interface with wrong mac resulting in finding none and trying to build new one which fails on validation